generators/arduino/leaphy_common.js

```diff
@@ -25,6 +25,18 @@ function getCodeGenerators(Arduino) {
     return null;
   };
 
+  Arduino.forBlock["leaphy_serial_available"] = function (block) {
+    Arduino.addSetup("serial", "Serial.begin(115200);", false);
+    var code = "Serial.available()";
+    return [code, Arduino.ORDER_ATOMIC];
+  };
+
+  Arduino.forBlock["leaphy_serial_read_line"] = function (block) {
+    Arduino.addSetup("serial", "Serial.begin(115200);", false);
+    var code = "Serial.readStringUntil('\\n')";
```

Comment:
```suggestion
    var code = "Serial.readStringUntil('\n')";
```

Reply:
In dit geval escape ik de \ voor de newline, om dit te voorkomen:
```
Serial.readStringUntil(
);
```
het idee is dat de code er zo uit ziet:
```
Serial.readStringUntil('\n');
```

Reply:
oh ja, duh, anders gaat JS er mee aan de haal..